### PR TITLE
docs(cn): fix configuration/optimization/noEmitOnErrors translate

### PR DESCRIPTION
**Pull request by Janlaywss**
这里有一处上下文语句不连贯，修复后阅读会更顺畅

_describe your changes..._

- [ ] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by Janlaywss** [comment]
@QC-L @dear-lizhihua @yuqingc

File: src/content/configuration/optimization.md [hunk 1]
```diff
@@ -164,8 +164,7 @@ module.exports = {
 
 `boolean`
 
-在编译出错时，使用 `optimization.noEmitOnErrors` 来跳过生成阶段(emitting phase)。这可以确保没有生成出错误资源。而 stats 中所有 assets 
-中的 `emitted` 标记都是 `false`。
+在编译出错时，使用 `optimization.noEmitOnErrors` 来跳过生成阶段(emitting phase)。这可以确保没有生成出错误资源，但是 stats 中所有 assets 属性中的 `emitted` 都会标记为 `false`。
```
Comment (QC-L) [inline src/content/configuration/optimization.md]:
```suggestion
在编译出错时，使用 `optimization.noEmitOnErrors` 来跳过生成阶段(emitting phase)。这可以确保没有生成出错误资源，
但是 stats 中所有 assets 属性中的 `emitted` 都会标记为 `false`。
```

Comment (QC-L) [reply src/content/configuration/optimization.md]: 原本换行是需要保留的~

**Comment by Janlaywss** [comment]
@QC-L 👍

**Review by QC-L — APPROVED** [review]

**Pull request merged**